### DISCUSSION TITLE
Rotation / Scale around point

### DISCUSSION THREAD

**Discussion by luckyde** [shader-park/shader-park-core#75] [Ideas]
Hi there, I would like to request if possible the ability to 
Rotate / Scale around a point , perhaps in this fashion
(rotate x , based off global position x 0 y : 1 z: 0.5
`rotateX(45*PI/180,vec3(0, 1, 0.5) );`

or around pivots , based off minimum x position, center y position, maximum z position
`rotateX(45*PI/180,vec3(bounds_x.min, bounds_y.centroid, bounds_z.max) );`
Same with scale functionality. Transform functionality wouldn't be needed as there's no benefit there
Hope that makes sense, thank you :)

**Comment by PWhiddy** [comment]
To scale or rotate around a point, you can displace with that point, use scaleShape or rotate functions, and then displace back by -point. You could encapsulate this into a function for convenience!
